packages/itwin/tree-widget/src/components/trees/stateless/imodel-content-tree/IModelContentTreeDefinition.ts

```diff
@@ -0,0 +1,701 @@
+    });
+    return [
+      {
+        fullClassName: "BisCore.Element",
+        query: {
+          ecsql: `
+            SELECT
+              ${await this._selectQueryFactory.createSelectClause({
+                ecClassId: { selector: "this.ECClassId" },
+                ecInstanceId: { selector: "this.ECInstanceId" },
+                nodeLabel: {
+                  selector: await this._nodeLabelSelectClauseFactory.createSelectClause({
+                    classAlias: "this",
+                    className: "BisCore.Element",
+                  }),
+                },
+                hasChildren: {
+                  selector: `
+                    IFNULL((
+                      SELECT 1
+                      FROM (
+                        SELECT Parent.Id ParentId FROM BisCore.Element
+                        UNION ALL
+                        SELECT ModeledElement.Id ParentId FROM BisCore.Model
+                      )
+                      WHERE ParentId = this.ECInstanceId
+                      LIMIT 1
+                    ), 0)
+                  `,
+                },
+                grouping: {
+                  byClass: true,
+                },
+                extendedData: {
+                  imageId: "icon-item",
+                },
+                supportsFiltering: true,
+              })}
+            FROM ${instanceFilterClauses.from} this
+            JOIN BisCore.Model m ON m.ECInstanceId = this.Model.id
+            ${instanceFilterClauses.joins}
+            WHERE
+              m.ECClassId IS NOT (BisCore.GeometricModel) AND
+              this.Parent IS NULL AND this.Model.Id IN (${modelIds.map(() => "?").join(",")})
+              ${instanceFilterClauses.where ? `AND ${instanceFilterClauses.where}` : ""}
+          `,
+          bindings: modelIds.map((id) => ({ type: "id", value: id })),
+        },
+      },
+    ];
+  }
+
+  private async createGroupInformationElementChildrenQuery({
+    parentNodeInstanceIds: groupIds,
+  }: DefineInstanceNodeChildHierarchyLevelProps): Promise<HierarchyLevelDefinition> {
+    return [
+      {
+        node: {
+          key: "ChildrenNode",
+          label: "Children",
+          extendedData: {
+            groupIds,
+            imageId: "icon-hierarchy-tree",
+          },
+          supportsFiltering: true,
+          processingParams: {
+            hideIfNoChildren: true,
+          },
+        },
+      },
+      {
+        node: {
+          key: "MembersNode",
+          label: "Members",
+          extendedData: {
+            groupIds,
+            imageId: "icon-group",
+          },
+          supportsFiltering: true,
+          processingParams: {
+            hideIfNoChildren: true,
+          },
+        },
+      },
+    ];
+  }
+
+  private async createChildrenNodeChildrenQuery({ parentNode, instanceFilter }: DefineCustomNodeChildHierarchyLevelProps): Promise<HierarchyLevelDefinition> {
```

Comment:
I'd rename this to `createGroupInformationElementChildElementsQuery`

Reply:
Renamed.